src/documentation/io/built-in.md

```diff
@@ -56,6 +60,7 @@ Consult the [Programming Guide I/O section]({{site.baseurl }}/documentation/prog
 <tr>
   <td>Python</td>
   <td>
+    <p>Beam Python supports GCS and local filesystems out of the box. Support for HDFS is in progress (<a href="https://issues.apache.org/jira/browse/BEAM-3099">BEAM-3099</a>).</p>
```

Comment:
Should we add Variant Call Format (VCF) (I think added in 2.3.0?) for Python?

Reply:
Done.